src/configuration/config.ts

```diff
@@ -19,14 +19,27 @@ const config: Config = {
   graphql: {
     debug: Env.isDev,
     introspection: Boolean(getEnv('INTROSPECTION_ENABLED')),
-    playgroundEnabled: false,
+    playground: {
+      settings: {
+        'request.credentials': 'include',
+      },
+    },
 
     cors: {
       credentials: true,
       originAsRegex: /https?:\/\/[a-z0-9\-_]+-fullstacks\.vercel\.app/,
       apolloSandboxOrigin: 'https://studio.apollographql.com',
     },
   },
+
+  auth: {
+    // https://github.com/expressjs/session
+    session: {
+      secret: getEnv('SESSION_SECRET', 'secret'),
+      resave: true,
+      saveUninitialized: true,
```

Comment:
why tru?

"Forces a session that is "uninitialized" to be saved to the store. A session is uninitialized when it is new but not modified. Choosing false is useful for implementing login sessions, reducing server storage usage, or complying with laws that require permission before setting a cookie. Choosing false will also help with race conditions where a client makes multiple parallel requests without a session." https://github.com/expressjs/session#saveuninitialized